canonical-web-and-design/juju.is
# Fix search page layout bugs

## Done

- Fix search page layout bugs
- Correct accent color of list items

## QA

- Check out this feature branch
- Run the site using the command `./run serve` and run with search API key
- View the site locally in your web browser at: https://juju.is/docs/search?q=asasas
- Verify tick icons are brand color
- Verify that reset icon in search box displays correctly


## Issue / Card

Fixes #81
